MatterControl.Printing/PrinterCommunication/Drivers/TCPIP/TcpipSerialPort.cs

```diff
@@ -200,9 +200,19 @@ public void Open()
 
 		private void LogInfo(string message)
 		{
-			ApplicationController.Instance.LogInfo(message);
+			// TODO: Reimplement messaging
+			// ApplicationController.Instance.LogInfo(message);
+			System.Diagnostics.Debugger.Break();
 		}
 
+		private void LogError(string message)
+		{
+			// TODO: Reimplement messaging
+			// ApplicationController.Instance.LogInfo(message);
+			System.Diagnostics.Debugger.Break();
+		}
```

Comment:
Create follow-up issue

Reply:
Issue created - MatterHackers/MCCentral#5763
Implement messaging system from SerialPort to listener